### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by imhassantariq**
This PR includes:

- UCSD theme for credentials service

image

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@imhassantariq please add screenshots for the changes.

**Comment by imhassantariq** [comment]
> @imhassantariq please add screenshots for the changes.

@tasawernawaz  Added in PR description

File: credentials/templates/base.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{% block title %}{% endblock title %}</title>
 
-    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="icon" type="image/x-icon" href="/static/images/favicon.ico" />
```
Comment (danialmalik) [inline credentials/templates/base.html]: What if we place the icon in the root of our theme app? That way we wouldn't have to change this edx repo.
What do you think?

**Comment by tasawernawaz** [comment]
@imhassantariq my instinct says he will ask for move logo to left-aligned.

File: credentials/templates/base.html (hunk 1 above)
Comment (imhassantariq) [reply credentials/templates/base.html]: It's not working.

**Comment by imhassantariq** [comment]
> @imhassantariq my instinct says he will ask for move logo to left-aligned.

@tasawernawaz Done.
image

File: credentials/templates/base.html (hunk 1 above)
Comment (HamzaIbnFarooq) [inline credentials/templates/base.html]:
```suggestion
    <link rel="icon" type="image/x-icon" href="{% static "images/favicon.ico" %}" />
```
can we do somthing like this?

Comment (imhassantariq) [reply credentials/templates/base.html]: Done

**Pull request merged**